docs/install.rst

```diff
@@ -90,6 +93,11 @@ Disable and remove the swapfile once the virtual enviroment is constructed::
   user@webserver:~$ sudo swapoff /tmp/swapfile
   user@webserver:~$ sudo rm /tmp/swapfile
 
+Custom Python environment
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Note that if you have configured any custom Python environment variables on your system, make sure they are properly (un)set when running `certbot-auto` and `sudo`. This could happen if you run `sudo zsh` as your user's `PYTHONPATH` will be preserved which might cause problems.
```

Comment:
In general descendant processes inherit the environment of their parent. If you run `sudo su`, you get a login shell that clears the environment and loads root's default environment (through the various mechanisms that do so). However, if you run `sudo bash` or `sudo zsh`, most of the environment gets kept (though this may differ depending on sudo permissions).